src/classes/LD_LeadConvertOverride_TEST.cls

```diff
@@ -91,15 +91,19 @@ private class LD_LeadConvertOverride_TEST {
         ext.getConList();
         system.assertEquals(2, ext.conList.size());
         system.assert(ext.lcStatuses.size() > 0);
-        system.assert(ext.leadConvStatus != null && ext.leadConvStatus != '');
         system.assertEquals(1, ext.listSOAccounts.size());
         system.assertEquals(1, ext.listSOAfflAccounts.size());
 
         // this selects the "New" option in the Contact box
         ext.strConId = 'NEW_CONTACT';
         ext.strAccountId = 'AUTO';
+
         Test.startTest();
-        ext.convertLeadNow();
+        for (SelectOption lcStatus : ext.lcStatuses) {
```

Comment:
I'm guessing the tests did this loop, because they didn't know what status value would actually convert the lead.  are you certain that convertLeadNow() will always succeed?